environments/stg/backend_lb_config.yaml

```diff
@@ -246,6 +246,18 @@ gateways:
      # Unspec
       - product: unspec
         component: service
+      - product: civil-damages
+        component: service-xui-staging
+      - product: civil-damages
+        component: service-camunda-staging
+      - product: civil-damages
+        component: ccd-xui-staging
+      - product: civil-damages
+        component: ccd-camunda-staging
+      - product: civil-damages
+        component: camunda-xui-staging
+      - product: civil-damages
+        component: camunda-camunda-staging
```

Comment:
how about ?
```suggestion
        component: camunda-staging
```

Reply:
I'd rather leave it as it is to keep it consistent and avoid confusion.